contents/blog/5-ways-to-improve-analytics-data.md

```diff
@@ -0,0 +1,133 @@
+---
+date: 2023-09-06
+title: 5 ways to improve your analytics data
```

Comment:
```suggestion
title: How to structure your analytics data so it's usable
```

I feel like this is the most "what it says on the tin" title for this. Talk of "foundations" is too abstract.